src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -898,10 +899,18 @@ export default class ElementWrapper extends Wrapper {
 				const all_dependencies = this.class_dependencies.concat(...dependencies);
 				const condition = block.renderer.dirty(all_dependencies);
 
-				block.chunks.update.push(b`
+				// If all of the dependencies are non-dynamic (don't get updated) then there is no reason
+				// to add an updater for this.
+				const any_dynamic_dependencies = all_dependencies.some((dep) => {
+					const v = this.renderer.component.var_lookup.get(dep);
+					return !v || is_dynamic(v);
```

Comment:
Rename variable `v` to something meaningful, such as `variable`.

Beyond that, this PR is great 👍

Reply:
Oops, completely missed that! Will push a commit to fix that